rsyncdconfig/config.go

```diff
@@ -20,6 +20,14 @@ type Module struct {
 	ACL  []string `toml:"acl"`
 }
 
+func NewModule(name string, path string, acl []string) Module {
```

Comment:
Thinking about it, I think it’s better to let users construct a `Module` by using a literal. That way, adding new fields doesn’t mean that we need to add a new variant of this function.

Reply:
Sure, but I think that makes validation more important (so we don't get modules with empty name and path). I was already a bit hesitant about this `NewModule()` not doing validation (therefore not having `error` return, which itself would've also felt.. ugly :D).

I added module validation to `rsyncd.NewServer(modules)`, WDYT?